content-src/selectors/selectors.js

```diff
@@ -68,10 +68,11 @@ module.exports.selectNewTabSites = createSelector(
     state => state.TopSites,
     state => state.FrecentHistory,
     state => state.History,
+    state => state.Highlights,
     selectSpotlight,
     state => state.Blocked
   ],
-  (TopSites, FrecentHistory, History, Spotlight, Blocked) => {
+  (TopSites, FrecentHistory, History, Highlights, Spotlight, Blocked) => {
```

Comment:
That's what I did originally, however, it produced an inferior highlights UX.

I did file a follow-up bug so we can iterate on this feature: #533